test/interop/C/multilocale/use_cstringArgAndReturn.ml-test.c

```diff
@@ -13,7 +13,8 @@ int main(int argc, char* argv[]) {
   const char* c = "The last of the strings";
   const char* d = takeAndReturn(c);
   take(d);
-
+  free((char*)b);
```

Comment:
We'll probably end up changing these to `chpl_free` eventually, when a wrapper for that function is supported on the client side?

Reply:
Yup!  But I figured it was better to free the memory than leave it leaking or leave a call that causes an error present.